deploy/stratos-base-images/Dockerfile.stratos-go-base.tmpl

```diff
@@ -0,0 +1,15 @@
+FROM {{BASE_IMAGE}}
```

Comment:
Rename to Dockerfile.stratos-go-build-base.tmpl

and update dependencies on this name.

Reply:
updated!